packages/clay-alert/package.json

```diff
@@ -21,7 +21,8 @@
     "compile": "babel -d lib/ src/ -s --ignore src/__tests__",
     "link": "npm link",
     "prepublish": "npm run soy && npm run compile",
-    "soy": "metalsoy --soyDeps '../../node_modules/clay-+(button|icon)/src/**/*.soy'"
+    "soy": "metalsoy --soyDeps '../../node_modules/clay-+(button|icon)/src/**/*.soy'",
+    "yarnLink": "yarn link"
```

Comment:
How about instead of adding `yarnLink` we just change `"link": "npm link"` to `"link": "yarn link"`?

Reply:
If nobody wants the option of linking with npm anymore I'm ok with that. @jbalsas ?

Reply:
we're killing `npm` usage for now, so yeah, just `link: yarn link` sounds fine